.jenkins/common_job_properties.groovy

```diff
@@ -168,6 +174,13 @@ class common_job_properties {
       scm(scm_schedule)
     }
 
+    // Enable triggering oneshots of pull requests. Users can comment the trigger phrase
+    // speficied in the postcommit job and have the job run against their PR to run
+    // tests not in the presubmit suite for additional confidence.
+    if (trigger_phrase != '') {
+      setPullRequestBuildTrigger(context, 'Jenkins PostCommit', '--none--', trigger_phrase)
```

Comment:
Well until we get the bug fixed there's no way to stop Jenkins from informing you where you can see build results :P I decided this functionality lived better in its own method -- it makes it clear in the individual jobs what is actually going on.